p/scripts/main.js

```diff
@@ -1089,19 +1089,29 @@ function init_stream(stream) {
 		el = ev.target.closest('.item.share > button[data-type="clipboard"]');
 		if (el) { // Clipboard
 			if (navigator.clipboard) {
-				navigator.clipboard.writeText(el.dataset.url);
+				navigator.clipboard.writeText(el.dataset.url)
+					.then(() => {
+						toggleClass(el, 'error');
+					})
+					.catch(e => {
+						console.log(e);
+						toggleClass(el, 'error');
+					});
 			} else {
 				// fallback, if navigator.clipboard is not available f.e. if access is not via https or localhost
 				const inputElement = document.createElement('input');
 				inputElement.value = el.dataset.url;
 				document.body.appendChild(inputElement);
 				inputElement.select();
-				document.execCommand('copy');
+				if (document.execCommand('copy')) {
```

Comment:
```suggestion
				if (document.execCommand && document.execCommand('copy')) {
```

To account for the fact that document.execCommand is obsoleted. But as Frenzie wrote, not super important